packages/tempo/public/bundle.index.html

```
@@ -0,0 +1,12 @@
+<script type="importmap">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)